apps/system/js/sim_lock_manager.js

```diff
@@ -159,6 +159,7 @@
 
     '_handle_airplanemode-enabled': function(evt) {
       this.simLockSystemDialog.close();
+      this._alreadyShown = false;
```

Comment:
Actually, this flag is saving a different status. This one is used for showing the first slot PIN dialog always before the second one. It was a quick fix to make it land to 2.1. I hope we can get rid of it soon as well..